support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.tsx

```diff
@@ -514,13 +494,14 @@ function PaperCheckoutForm(props: PropTypes) {
 							country={props.country}
 							isTestUser={props.isTestUser}
 							submitForm={props.submitForm}
-							allErrors={[
-								...props.billingAddressErrors,
-								...props.deliveryAddressErrors,
-								...props.formErrors,
-							]}
+							allErrors={
+								[
+									...props.billingAddressErrors,
+									...props.deliveryAddressErrors,
+									...props.formErrors,
+								] as Array<FormError<FormField>>
```

Comment:
Type casting here to fit expected prop type

Reply:
Can we remove this cast by updating the props types? i.e

```ts
		billingAddressErrors: Array<FormError<FormField>>;
		deliveryAddressErrors: Array<FormError<FormField>>;
```

Reply:
The props types are inferred from state so I reckon changing them would require a bigger refactor

Reply:
I had a go at this one locally and I think it works out. The type of e.g `billingAddressErrors` is here: https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/components/subscriptionCheckouts/address/addressFieldsStore.ts#L36 which is actually the required type for this array!

Reply:
What was it that you'd tried locally?

Reply:
ooh I missed the refactor to use the connected props thing. Let me have a quick go again...

Reply:
Okay, yeah not worth it. There's a bigger refactor in there.